webapp/src/components/AssessmentsPage.tsx

```diff
@@ -0,0 +1,410 @@
+import {
+  Box,
+  Chip,
+  Container,
+  Stack,
+  Button,
+  Paper,
+  TableRow,
+  TableHead,
+  TableContainer,
+  TableCell,
+  TableBody,
+  Table,
+  Tabs,
+  Tab,
+} from '@mui/material';
+
+import React, { useState } from 'react';
+import Badge, { BadgeProps } from '@mui/material/Badge';
+import { styled } from '@mui/material/styles';
+
+import CheckOutlinedIcon from '@mui/icons-material/CheckOutlined';
+import DoneAllOutlinedIcon from '@mui/icons-material/DoneAllOutlined';
+import ScheduleOutlinedIcon from '@mui/icons-material/ScheduleOutlined';
+import CancelOutlinedIcon from '@mui/icons-material/CancelOutlined';
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined';
+import UpcomingOutlinedIcon from '@mui/icons-material/UpcomingOutlined';
+import LockClockOutlinedIcon from '@mui/icons-material/LockClockOutlined';
+import AssessmentIcon from '@mui/icons-material/Assessment';
+import { formatDateTime } from '../helpers/dateTime';
```

Comment:
There needs to be a line break above line 30. Again, see my previous comment; we usually keep a line break between external imports (that is to say, imports from other packages) and imports from other files in the project.